assets/node-exporter/prometheus-rule.yaml

```diff
@@ -41,7 +41,7 @@ spec:
         (
           node_filesystem_avail_bytes{job="node-exporter",fstype!=""} / node_filesystem_size_bytes{job="node-exporter",fstype!=""} * 100 < 15
         and
-          predict_linear(node_filesystem_avail_bytes{job="node-exporter",fstype!=""}[6h], 4*60*60) < 0
+          predict_linear(node_filesystem_avail_bytes{job="node-exporter",fstype!=""}[6h], 2*60*60) < 0
```

Comment:
Updating expressions isn't in the scope of this PR, we should stick to only fixing the severity.